docs/releases/overview.md

```diff
@@ -18,7 +18,39 @@ See the full release notes on [GitHub ↗](https://github.com/vegaprotocol/vega/
 
 
-## Pre-release version v0.75.0 preview.3  | 2024-03-06
+## Pre-release versions v0.75.0-preview.4 and v0.75.0-preview.5 (combined)  | 2024-03-13
+This version was released to the Vega testnet on 13 March 2024.
+
+### Bug fixes
+
+- The `ListTransfers` API fails when given a cursor resulting in the API not always constructing valid queries and returns an internal error. This has been resolved in [issue 10837 ↗](https://github.com/vegaprotocol/vega/issues/10837).
+- During testing points markets in market-sim using numerous auction triggers, markets were entering auctions for the correct duration but being extended for incorrect durations. A fix has been introduced to only break single bound on auction exit. this was resolved in [issue 10823 ↗](https://github.com/vegaprotocol/vega/issues/10823).
+- It was identified that the command line wallet leaves terminal input invisible upon Ctrl-C at the wrong time. This has now been fixed in [issue 10055 ↗](https://github.com/vegaprotocol/vega/issues/10055).
+- When preparing for the 0.75 incentive on testnet it was found that the addition of the check so that the mark price calculation can trigger a price auction was causing the book-keeping of enter/leaving auctions for perpetuals to become out of sync. The protocol now tells the perpetual that it is leaving when we call `as.Left()`. It is then updated again to enter an auction if the price-monitoring is triggered. The perpetual then just handles this zero length interval. This was fixed in [issue 10858 ↗](https://github.com/vegaprotocol/vega/issues/10858).
+- After the deployment of 0.74.9 validators that use Sentry nodes were reporting a high RAM usage. It was found that in some cases nodes were submitted in many duplicate `NodeVoteCommand` commands. The window during which the node is expecting to see its own vote is currently a hard-coded to `v.lastSentVote.Add(10 * time.Second)`. This has been resolved by adding a config value and setting it to a higher value than the current hard-coded value. As a result the number of failed transactions (node votes) due to duplicate votes or invalid resource ID should decrease, along with RAM usage. This has been addressed in [issue 10862 ↗](https://github.com/vegaprotocol/vega/issues/10862).
+- A bug was found in the `EstimatePosition` collateral increase estimate API. It was found that the estimate returned from API and the actual difference between the general account balance is significantly different. This has now been addressed and the API estimate and actual difference is now the same. This was fixed in [issue 10852 ↗](https://github.com/vegaprotocol/vega/issues/10852).
```

Comment:
```suggestion
- The `ListTransfers` API would fail when given a cursor, resulting in the API not always constructing valid queries and returning an error. This has been resolved in [issue 10837 ↗](https://github.com/vegaprotocol/vega/issues/10837).
- In testing using numerous auction triggers, markets were entering auctions for the correct duration but being extended for incorrect durations. A fix has been introduced to only break single bound on auction exit. This was resolved in [issue 10823 ↗](https://github.com/vegaprotocol/vega/issues/10823).
- It was identified that the command line wallet leaves terminal input invisible upon Ctrl-C at the wrong time. This has now been fixed in [issue 10055 ↗](https://github.com/vegaprotocol/vega/issues/10055).
- When preparing for the 0.75 testnet incentive, it was found that the new check for mark price calculations triggering a price auction was causing the bookkeeping of enter/leaving auctions for perpetual markets to become out of sync. The protocol now tells the market that it is leaving auction, and is then updated again to enter an auction if price monitoring is triggered. The market will be able to handle this zero-length interval. This was fixed in [issue 10858 ↗](https://github.com/vegaprotocol/vega/issues/10858).
- After the deployment of 0.74.9, validators that use Sentry nodes reported high RAM usage. In some cases nodes were submitting duplicate node vote commands. The window during which the node is expecting to see its own vote was hard-coded to `v.lastSentVote.Add(10 * time.Second)`. This has been resolved by adding a config value and setting it to a higher value than the hard-coded value. As a result the number of failed node votes due to duplicate votes or invalid resource IDs should decrease, along with RAM usage. This has been addressed in [issue 10862 ↗](https://github.com/vegaprotocol/vega/issues/10862).
- A bug was found in the `EstimatePosition` API with regards to collateral increase. The estimate returned from the API and the actual difference between the general account balance was significantly different. This has now been addressed and the API estimate and actual difference are now the same. This was fixed in [issue 10852 ↗](https://github.com/vegaprotocol/vega/issues/10852).
```